src/Stryker.CLI/Stryker.CLI.UnitTest/StrykerCLITests.cs

```diff
@@ -109,5 +109,21 @@ public void StrykerCLI_ShouldPassLogFileArgumentsToStryker(string argName)
 
             mock.Verify(x => x.RunMutationTest(It.Is<StrykerOptions>(o => o.LogOptions.LogToFile == true)));
         }
+
+        [Theory]
+        [InlineData("--timeoutMS")]
+        [InlineData("-t")]
+        public void StrykerCLI_ShouldPassTimeoutArgumentsToStryker(string argName)
```

Comment:
I'm missing the `act` part of this test name: `StrykerCLI_WithTimeoutArgument_ShouldPassTimeoutToStryker`